WCS/build.properties

```diff
@@ -1,3 +1,8 @@
 project_name = Sample
 project_base = src
 project_target = target
+
+# All Environment configurations below
+
+DEPLOY_ENV = "DEV"
```

Comment:
Add more parameters for QA, UAT environment variables, and respective server information.